frontend/public/style/ancillary/_bootstrap-residual.scss

```diff
@@ -42,19 +29,10 @@ code, kbd, pre, samp {
 
 
 hr {
```

Comment:
Seems like a good opportunity to remove this generic element rule, replace all `<hr>` instances with `<Divider className="co-divider" />` and corresponding new CSS rule for margins?

Reply:
Switched out `<hr>` instances for `<Divider className="co-divider--with-margins" />` Found several others.

Reply:
It looks like you forgot to push the changes?

Reply:
Updates pushed